### PR TITLE
Update fac services (repartitioning IOCs)

### PR DESCRIPTION
**Pull request by xresende**
A new IOC partition into docker services

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -106,17 +106,35 @@ service-facs-si-ps-trims-c1234-stop:
...
+
+service-facs-as-ap-opticscorr-stop:
+	cd services; docker stack rm facs-as-ap-opticscorr
+
+service-facs-as-ap-opticscorr-start:
+	cd services; docker stack deploy -c docker-stack-as-ap-opticscorr.yml facs-as-ap-opticscorr
 
-service-facs-as-ap-1-start:
-	cd services; docker stack deploy -c docker-stack-as-ap-1.yml facs-as-ap-1
+service-facs-as-ap-machshift-stop:
```
Comment (xresende) [inline Makefile:127]: @fernandohds564 , @anacso17 What about moving to 'macshift' ? I find 'machshift' very non-intuitive. I always have 'MAC meeting' in my head, for example.

Comment (fernandohds564) [reply Makefile:127]: I think we can change it @xresende . I don't remember why we chose `mach` over `mac`.

**Review by fernandohds564 — APPROVED** [review]

File: services/docker-stack-as-ap-machshift.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+services:
+  tb-ps:
+    image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:2020-02-26
+    command: bash -c 'source /etc/profile && /ioc-apps/as-ap-macshift.bash'
```
Comment (anacso17) [inline services/docker-stack-as-ap-machshift.yml:5]: typo here? /ioc-apps/as-ap-macshift.bash --> /ioc-apps/as-ap-mac**h**shift.bash

File: services/docker-stack-as-ap-opticscorr.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+services:
+  tb-ps:
+    image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:2020-02-26
+    command: bash -c 'source /etc/profile && /ioc-apps/as-ap-opticscorr.bash'
+    deploy:
+      placement:
+        constraints:
+          - node.hostname != CA-RaTim-CO-IOCSrv
```
Comment (carneirofc) [inline services/docker-stack-as-ap-opticscorr.yml:9]: This server is responsible for the timing IOCs.
Is this a typo, or we are actively avoiding it for some reason?

**Review by carneirofc — COMMENTED** [review]
The constraint `node.hostname != CA-RaTim-CO-IOCSrv` is expected?

**Review by murilobalves — APPROVED** [review]

File: services/docker-stack-as-ap-opticscorr.yml (hunk 3 above)
Comment (xresende) [reply services/docker-stack-as-ap-opticscorr.yml:9]: it is not a type. I understand from talking to you guys we wanted to run fac services only in dell nodes, no? can we include CA-RaTim-CO-IOCSrv ?

File: services/docker-stack-as-ap-machshift.yml (hunk 2 above)
Comment (xresende) [reply services/docker-stack-as-ap-machshift.yml:5]: yes, this typo was still lurking. thanks.

**Review by fernandohds564 — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

File: services/docker-stack-as-ap-opticscorr.yml (hunk 3 above)
Comment (xresende) [reply services/docker-stack-as-ap-opticscorr.yml:9]: CA-RaTim-CO-IOCSrv IS a dell server. issue understood. I will comment this constraint.

**Pull request merged**